dongilbert/mautic
# Fix css conflict and other fixes

Please answer the following questions:

| Q | A |
| --- | --- |
| Bug fix? | Y |
| New feature? | N |
| BC breaks? | N |
| Deprecations? | N |
| Fixed issues |  |
## Description

Turns out that assets in directories under Assets/css or Assets/js are automatically loaded. This lead to the popup/ assets conflicting with Mautic's (changed link color, etc). 

This PR restricts auto-loading assets to the of `Assets/(css|js)` root or `Assets/(css|js)/(app|libraries)` sub folders.
## Steps to reproduce the bug (if applicable)

Apply the channel push PR and view Mautic. All the links will be the wrong color.
## Steps to test this PR

Apply this PR and the NotificationBundle/Assets/popup assets should not load but assets 

PS - I fixed a few other issues I came across while working on other things. See commits.
